cypress/e2e/onboarding.cy.ts

```diff
@@ -10,6 +10,8 @@ describe('Onboarding', () => {
                 })
             )
         )
+
+        cy.intercept('/api/billing-v2/', { fixture: 'api/billing-v2/billing-v2-unsubscribed.json' })
     })
```

Comment:
Is there a good way to set has_completed_onboarding_for? 
I tried intercepting `api/projects/@current`, which is what teamLogic calls to update the currentTeam info, but it didn't seem to work reliably.

Reply:
Setting this to false would let me click to a product intro page from the side bar instead of having to navigate to the url directly.

Reply:
Ok, I can set this by overriding the app context value. 

The annoying thing is now the feature flag intercept in this test doesn't work reliably. It seems to be ignored in favor of the one in the e2e.ts file so I've added the product-intro-pages flag there...not sure why this is so annoying.